misc/testnnp/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM fedora:latest
+ADD testnnp /root/testnnp
```

Comment:
I don't think this is a good idea to check in a binary.  Can you compile it in the Dockerfile instead?  Or build it earlier in the Makefile?